### PR TITLE
Added pause button for mobile

### PR DESCRIPTION
**Pull request by Andrea1141**
Added a pause button for mobile, as requested in #31.

### CODE REVIEW

File: scenes/gameplay/pause-layer/pause-layer.gd [hunk 1]
```diff
@@ -32,3 +37,9 @@ func _on_Main_Menu_pressed():
 	Game.change_scene("res://scenes/menu/menu.tscn", {
 		'show_progress_bar': false
 	})
+
+
+func _on_PauseButton_pressed():
+	resume_option.grab_focus()
+	get_tree().paused = true
+	pause.show()
```
Comment (davcri) [inline scenes/gameplay/pause-layer/pause-layer.gd:45]: Can we simply call `pause_game()` and avoid duplication? What do you think?

**Comment by davcri** [comment]
Hi @Andrea1141 thanks for the pull request! I left a comment let me know what do you think. 

Also can you make the pull request to the `dev` branch and not the `main` branch? I don't know if GitHub allows to change the branch from here or if we need to create a new PR.

File: scenes/gameplay/pause-layer/pause-layer.gd (hunk 1 above)
Comment (Andrea1141) [reply scenes/gameplay/pause-layer/pause-layer.gd]: Yes, it's my fault, I didn't see the `pause_game()` function.

**Comment by Andrea1141** [comment]
@davcri I added the `pause_game()` function to avoid duplication but I can't change branch, so I'm creating another pull request.

**Pull request closed without merging**